src/app/static/src/tests/integration/downloadResults.itest.ts

```diff
@@ -126,6 +126,6 @@ describe(`download results actions integration`, () => {
             expect(commit.mock.calls[1][0]["payload"].status).toBe("MISSING");
             done()
 
-        }, 2100)
+        }, 3100)
```

Comment:
Making this timeout consistent with the others - this was failing for me locally otherwise.